packages/blockchain-wallet-v4-frontend/src/data/prices/slice.ts

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable no-param-reassign */
```

Comment:
i disabled this rule globally in `development` branch.  merge latest development into your branch and then you can just delete this line.